### PR TITLE
Bump upstream and change state schema

### PR DESCRIPTION
**Pull request by 3alpha**

### CODE REVIEW

**Comment by Pol-Lanski** [comment]
Blocked for shit description provided.

File: entrypoint.sh [hunk 1]
```diff
@@ -31,19 +31,4 @@ esac
...
-  --http.corsdomain "*" \
-  --http.vhosts "*" \
-  --syncmode ${SYNCMODE:-snap} \
-  --ws \
-  --ws.origins "*" \
-  --ws.addr 0.0.0.0 \
-  --port ${P2P_PORT} \
-  --metrics \
-  --metrics.addr 0.0.0.0 \
-  --authrpc.addr 0.0.0.0 \
```
Comment (dappnodedev) [inline entrypoint.sh:45]: Don't you need to add the `authrpc` settings to the envs too?

**Review by dappnodedev — CHANGES_REQUESTED** [review]

File: docker-compose.yml [hunk 2]
```diff
@@ -1,17 +1,26 @@
...
+      - GETH_HTTP_VHOSTS=*
+      - GETH_SYNCMODE=snap
+      - GETH_WS=true
+      - GETH_WS_ADDR=0.0.0.0
+      - GETH_PORT=30403
+      - GETH_METRICS=true
+      - GETH_METRICS_ADDR=0.0.0.0
+      - GETH_STATE_SCHEME=path
     ports:
       - "30403:30403/tcp"
```
[the remark this answers is not in the document]
Comment (3alpha) [reply docker-compose.yml:25]: I left that just to preserve backward compatibility. Even better would be that we do not rename those vars, so I will fix that.

File: entrypoint.sh (hunk 1 above)
Comment (3alpha) [reply entrypoint.sh:45]: Yeah, but those are set in Dockerfile.

Comment (dappnodedev) [reply entrypoint.sh:45]: Okay, you are right

**Pull request closed without merging**